src/components/Modal/Modal.tsx

```diff
@@ -306,8 +311,13 @@ class Modal extends React.Component<CombinedProps, State> {
 
     const animated = !instant;
 
+    const trigger = activator
+      ? React.cloneElement(activator, {ref: this.activatorRef})
```

Comment:
As per our discussion, it looks like you can wrap the activator in a `div` with a `ref` and use `focusFirstFocusableNode` to focus the button. As a result, there should be no need to add the `forwardRef` to the Button component :tada: